golos.publication/golos.publication.cpp

```diff
@@ -160,13 +154,6 @@ void publication::create_message(
         curators_prcnt = curators_prcnt_param.min_curators_prcnt;
     }
 
-    if (parent_id.author) {
```

Comment:
This removing  is OK, because block list doesn't contain all records.
It can be restored later, after syncing of pin/block lists with ee-genesis.